keps/prod-readiness/sig-api-machinery/4460.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4460
+alpha:
+  approver: "@deads2k"
```

Comment:
Feel free to switch to me - I already reviewed it anyway.

Reply:
Done